recipes/serapeum_utils/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "serapeum_utils" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Serapieum-of-alex/serapeum_utils/archive/{{ version }}.tar.gz
+  sha256: 657d84d21ffaba7d785d39ab296d7130618c9179d6db5bed831a3728ca379403
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python  >=3.9,<3.11
+    - pip >=22.3.1
+    - poetry >=1.2.2
+  run:
+    - python  >=3.9,<3.11
+    - pip >=22.3.1
+
+test:
+  imports:
+    - serapeum_utils
+
+about:
+  home: https://github.com/Serapieum-of-alex/serapeum_utils
+  license:  GPL-3.0-only
+  license_family: GPL
+  license_file: LICENSE.md
+  summary: python utility package
+  description: |
+    python utility package (decorators handling functions with too many parameters).
```

Comment:
Kind of cryptic, no? Are these intended to the general public or  are they custom tools?

Reply:
they take a dictionary with each key is the parameter of the function and the value is another key
`initial_args= dict(
dx={"default": 500, "type": int}), 
dto={"default": 60, "type": int}
`
and use the @class_method_parse(initial_args) decorator to initialize and check the type of your input parameters
and other function to save and load objects as pickle and another progress bar function , so different utility function

Reply:
@ocefpaf any update?

Reply:
@conda-forge/staged-recipes not sure what to do here

Reply:
@conda-forge-admin, @conda-forge/help-python can you check please these have been like this for 10 days and no one want to check

Reply:
@MAfarrag your info about this package looks like is an in-house tool not something for the general public. I'm not against merging this but is it worth to use conda-forge's CI resources, online storage space, and infrastructure for something that could be just a local module in your workflow?

Reply:
the functionality is very basic at the moment cause I choose to add it to conda-forge while it is still simple but it will grow in the future, but as I explained it has four or five functions that handle functions with too many parameters, LL Packages serve a certain purpose and this package at the moment handles function parameters. maybe the package name does not reflect that is the main purpose of the package at the moment.

I will still ask to add this package to conda-forge please @ocefpaf if you decide not to merge it state your decision so someone else check it for me. I do not want to wait a lot I have already explained all the details.

Thanks

Reply:
>  I do not want to wait a lot I have already explained all the details.

OK. Then. I'll let someone else review this. @isuruf or @BastianZim if you have some time do you mind taking a look at this PR?

Reply:
Also not against merging it but a better description of what it does would be good. Please add some concise info on what it does in the description here and upstream and then I'd be happy to merge it.

Reply:
@BastianZim can you please check, I have changed the description.